src/libs/sfdk/sfdk.pro

```diff
@@ -41,3 +41,12 @@ HEADERS += \
     virtualmachine.h \
     virtualmachine_p.h \
     vmconnection_p.h \
+
+isEmpty(SFDK_OS_VARIANT):  SFDK_OS_VARIANT = Sailfish OS
+isEmpty(SFDK_SDK_VARIANT): SFDK_SDK_VARIANT = Sailfish SDK
+
+DISTFILES += $$PWD/sfdk_version_p.h.in
+QMAKE_SUBSTITUTES += $$PWD/sfdk_version_p.h.in
+
+INCLUDEPATH += \
+    $$IDE_BUILD_TREE/src/libs/sfdk # for "sfdk_version_p.h" in case of actual build directory
```

Comment:
`INCLUDEPATH += $$OUT_PWD` should work. If so, please stick it to the previous lines and it can go without comment then :)